airflow/providers/amazon/aws/operators/emr.py

```diff
@@ -742,10 +742,21 @@ def __init__(
         waiter_max_attempts: int | None = None,
         waiter_delay: int | None = None,
         waiter_countdown: int | None = None,
-        waiter_check_interval_seconds: int = 60,
+        waiter_check_interval_seconds: int,
         deferrable: bool = conf.getboolean("operators", "default_deferrable", fallback=False),
         **kwargs: Any,
     ):
+        if waiter_check_interval_seconds:
+            warnings.warn(
+                "The parameter waiter_check_interval_seconds has been deprecated to "
+                "standardize naming conventions.  Please use waiter_delay instead.  In the "
+                "future this will default to None and defer to the waiter's default value.",
+                AirflowProviderDeprecationWarning,
+                stacklevel=2,
+            )
+            waiter_delay = waiter_check_interval_seconds
```

Comment:
I dont think you should do that, you might override `waiter_delay` set by the user.

```suggestion
```